TEMPLATE/etc/httpd/conf.d/manageiq-external-auth-openidc.conf.erb

```diff
@@ -2,15 +2,36 @@ LoadModule auth_openidc_module modules/mod_auth_openidc.so
 ServerName https://<%= miq_appliance %>
 LogLevel   warn
 
-OIDCProviderMetadataURL  <%= oidc_provider_metadata_url %>
-OIDCCLientID             <%= oidc_client_id %>
-OIDCClientSecret         <%= oidc_client_secret %>
-OIDCRedirectURI          https://<%= miq_appliance %>/oidc_login/redirect_uri
-OIDCCryptoPassphrase     sp-cookie
-OIDCOAuthRemoteUserClaim username
+OIDCProviderMetadataURL            <%= oidc_provider_metadata_url %>
+OIDCCLientID                       <%= oidc_client_id %>
+OIDCClientSecret                   <%= oidc_client_secret %>
+OIDCRedirectURI                    https://<%= miq_appliance %>/oidc_login/redirect_uri
+OIDCCryptoPassphrase               sp-cookie
+OIDCOAuthRemoteUserClaim           username
+OIDCOAuthClientID                  <%= oidc_oauth_client_id %>
+OIDCOAuthClientSecret              <%= oidc_oauth_client_secret %>
```

Comment:
are OIDCOAuthClientID and OIDCAuthClientSecret required for the introspection ?  Don't they default to OIDCClientID and OIDCClientSecret ?   

If required, I'd vote to replace above oidc_oauth_client_id/_secret to the same as lines 6,7 just the oidc_cient_id, oidc_client_secret.  I don't think we need 2 sets of id/secret for the same OIDC client for a default configuration.

Reply:
Thank you @abellotti. I thought they were both needed. I'll double check.

Reply:
@abellotti 

I double checked and surprisingly OIDCOAuthClientID and OIDCAuthClientSecret are required for the introspection. They do not default to OIDCClientID and OIDCClientSecret.

Reusing the oidc_client_id/_secret tags in place of oidc_oauth_client_id/_secret as you have suggested.

Thank you!